static/src/javascripts/projects/commercial/modules/messenger/scroll.js

```diff
@@ -147,8 +146,4 @@ const onMessage = (respond: any, start, iframe: ?Element): void => {
     }
 };
 
-register('scroll', onMessage, {
-    persist: true,
-});
-
-export { addScrollListener, removeScrollListener, reset };
+export { addScrollListener, removeScrollListener, reset, onMessage };
```

Comment:
Again, let's move `addScrollListener`, `removeScrollListener` and `reset` to an export named `_` to show that they are only exposed for tests.